### ISSUE TITLE
Cannot add multiple artists with same name

### ISSUE THREAD

**Issue by archer0t8** [lidarr/Lidarr#320]
### System Information/Logs

**Lidarr Version:** 0.2.0.371

**Operating System:** Windows 7

**.net Framework (Windows) Version:** 4.6.1

**Log File Entry:** 
18-4-23 14:40:49.5|Warn|LidarrErrorPipeline|Invalid request Validation failed: 
 -- Path is already configured for another artist

### Additional Information

Looks like there is a bug related to adding multiple artists of the same name. I'm finding that I can't add Alice Cooper, the artist (mbid: ee58c59f-8e7f-4430-b8ca-236c4d3745ae) since I already have Alice Cooper, the group (mbid: 4d7928cd-7ed2-4282-8c29-c0c9f966f1bd); both are added under Alice Cooper and default to the path %ROOT%/Alice Cooper.

On the Add Artist interface there isn't a way to change the path for the artist; just the %ROOT% folder, so there is no way to directly fix this issue.

Current workaround is to add one artist and then change the folder it points to, then add the second artist.

### Description of request and what problem are you looking to solve?

Perhaps a way to change the artist folder from the Add Artist interface would allow for a simple fix of the issue?

**Comment by Qstick** [comment]
I assume you are opposed to separating into two separate artist folders in this case?

**Comment by archer0t8** [comment]
I would prefer not to separate, but currently that isn't even an option.

I tried to add them separately, but since they both try and default to the same path, the second artist cannot be added since that path is already in use for the first artist. And, there is no way to change the path for that one artist from within the interface from the default (ie. I can't change the /artist part of the path when adding an artist from whatever Lidarr wants to default it to)

**Comment by Qstick** [comment]
Currently, you would have to make a new artist path "music\Alice Cooper (group)" then Lidarr would see two different paths and you could assign an artist to each one.

**Comment by archer0t8** [comment]
Right, and that's what I ended up doing.

But wouldn't it make more sense to give me the ability to allow me to do that from within the program interface when adding an artist, and to at least tell me what the issue is? It didn't give any errors; I had to go log digging to figure out what the issue was.

**Comment by Qstick** [comment]
Yes, I'm agree with you that this could be a lot better. Was just offering up a temporary solution if you hadn't already done it.
